Tasks/DotnetCLIRestore/restorecommand.ts

```diff
@@ -0,0 +1,306 @@
+import * as fs from "fs";
+import * as ltx from "ltx";
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as Q from 'q';
+import * as auth from 'packaging-common/nuget/Authentication';
+import { NuGetConfigHelper2 } from 'packaging-common/nuget/NuGetConfigHelper2';
+import * as ngRunner from 'packaging-common/nuget/NuGetToolRunner2';
+import * as path from 'path';
+import { IExecOptions } from 'azure-pipelines-task-lib/toolrunner';
+import * as request from 'request';
+
+export async function run(): Promise<void> {
+    const buildIdentityDisplayName: string = null;
+    const buildIdentityAccount: string = null;
+
+    try {
+        let packageName: string = tl.getInput("packageName");
+        let packageVersion: string = tl.getInput("version") || "";
+
+        if (!packageName)
+        {
+            return;
+        }
+        else {
+            packageName = packageName.split("/")[1];
+        }
+
+        let endpointNames = tl.getDelimitedInput("externalEndpoints", ',');
```

Comment:
this input was optional so either make is mandatory or check for null here

Reply:
Made this a mandatory input.